src/App.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import './App.scss';
```

Comment:
according to the style guide, import styles should be at the bottom of the imports list